alicloud/service_alicloud_ecs.go

```diff
@@ -766,3 +766,36 @@ func (s *EcsService) QueryInstanceAllDisks(id string) ([]string, error) {
 	}
 	return ids, nil
 }
+
+func (s *EcsService) WaitUntilSnapshotCreatingCompeleted(snapshotId string) error {
+	for {
+		snapshot, err := s.DescribeSnapshotById(snapshotId)
+		if err != nil {
+			return err
+		}
+		if snapshot.Status == string(SnapshotCreatingAccomplished) {
+			return nil
+		}
+		if snapshot.Status == string(SnapshotCreatingFailed) {
+			return fmt.Errorf("create snapshot failed")
+		}
+
+		time.Sleep(DefaultIntervalShort * time.Second)
+	}
+}
```

Comment:
Please re-build the `WaitUntilSnapshotCreatingCompeleted ` according to the other waiting method, like `WaitForPvtzZone`.